ADR/0044-spdx-support.md

```diff
@@ -0,0 +1,279 @@
+|                                 |     {                                                      |
+|                                 |          spdxElementId = `<A-BUILDER-IMAGE-ID>`,           |
+|                                 |          relationshipType=BUILD_TOOL_OF,                   |
+|                                 |          relatedSpdxElement=`<ROOT-PACKAGE>`         |
+|                                 |     }                                                      |
+|                                 | ]                                                          |
+
+*Explanation: Root document `DESCRIBES` `ROOT-PACKAGE` element which represents the container itself. `BUILDER-IMAGE-ID` represents the builder image which was used to build the container. The relationship type `BUILD_TOOL_OF` is used to express that the builder image was used to build the container image.*
+
+#### Metadata.tools
+The CycloneDX metadata.tools sub attributes that we are mostly interested in are the vendor and name elements. Information about the creation of the SPDX document can be stored into creationInfo. CreationInfo.creators element is basically a list of strings. There’s a vague specification ([here](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#68-creator-field)]) about how it should be structured in the standard. Strings should be formatted in the following way: `<Attribute>: <Value>`. For example vendor should be stored as `Vendor: <vendor>`. Redommendation is to use only Tool as vendor can be misinterpreted as vendor of the SBOM not the tool which created it.
+
+| CyloneDX Attribute                             | SPDX Attribute                                    |
+|------------------------------------------------|---------------------------------------------------|
+| Metadata.tools = [{“vendor”: “X”, “name”: “Y”] | CreationInfo.creators = [“Tool: Y”]  |
+
+#### Metadata.component
+Metadata component describes component which is the component which whole SBOM is related to. For example If SBOM describes internal components and dependencies of a container image, this component should represent the container image itself. In SPDX, a package which is equivalent to this component is root package (see [root-packages](#root-packages) to have idea how syft handles this package). This package is in relationship `SPDXRef-ROOT` `DESCRIBES` `SPDXRef-RootPackage`. If there's no cycloneDX metadata.component attribute, SPDX SBOM already includes this "root package" and shouldn't be changed. If there is metadata.component attribute, existing root package should be replaced with the new one representing the SBOM document. All SPDX IDs in relationships refering to the old root package should be replaced with SPDX ID of the new root package. Example:
+We run syft/cachi2 on source directory of a project which should be build into container. Generated SBOM contains
+```json
+{
+  "SPDXID": "SPDXRef-DOCUMENT",
+  ...
```

Comment:
nitpick: the JSON syntax errors are distracting (especially in the un-rendered source, which is entirely red now). You could avoid them by using `jsonc` for the syntax and using comments
```suggestion
```jsonc
{
  "SPDXID": "SPDXRef-DOCUMENT",
  // ...
```